docs/SUMMARY.md

```diff
@@ -16,6 +16,7 @@
   * [Feature retrieval](getting-started/concepts/feature-retrieval.md)
   * [Point-in-time joins](getting-started/concepts/point-in-time-joins.md)
   * [Registry](getting-started/concepts/registry.md)
+  * [Permissions](getting-started/concepts/permissions.md)
```

Comment:
nit: I think "Role Based Access Controls (RBAC)" may be more explicit

Reply:
@tokoko do you agree with this naming? If yes, we can udate the docs accordingly (this and the other related comments)

Reply:
@dmartinol If we add some other type of policy (meaning other than `RoleBasedPolicy`), we would technically be going beyond rbac, right? From that point of view more generic title (Permission or Access Control) seems more appropriate.

Reply:
@tokoko yes, that was the original idea, even if I'm not sure anyone would ever want to do that. Anyway, it's open-source for a good reason 😉

Reply:
Makes sense.